### PR TITLE
saucelabs-espresso-app-automate 0.1.4

### PR DESCRIPTION
**Pull request by moatazeldebsy**
TagCheck

### New Pull Request Checklist

*Please mark the points which you did / accept.*

- [x] __I will not move an already shared step version's tag to another commit__
- [x] I read the [Step Development Guideline](https://github.com/bitrise-io/bitrise/blob/master/_docs/step-development-guideline.md)
- [x] I have a test for my Step, which can be run with `bitrise run test` (in the step's repository)
- [x] I did run `bitrise run audit-this-step` (in the step's repository - note, if you don't have this workflow in your `bitrise.yml`, [you can copy it from the step template](https://github.com/bitrise-steplib/step-template/blob/master/bitrise.yml).)
- [x] I read and accept the [Abandoned Step policy](https://github.com/bitrise-io/bitrise-steplib#abandoned-step-policy)

### CODE REVIEW

**Comment by moatazeldebsy** [comment]
one more `PR` because I changed in the `arguments` and the `description`.

File: steps/saucelabs-espresso-app-automate/0.1.4/step.yml [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+    is_sensitive: true
+    summary: Your username for saucelabs
+    title: SauceLabs username
+  saucelabs_username: null
+- opts:
+    description: ""
+    is_expand: true
+    is_required: true
+    is_sensitive: true
+    summary: Your username for saucelabs
```
Comment (godrei) [inline steps/saucelabs-espresso-app-automate/0.1.4/step.yml]: This input is for the `access key`, pls fix the summary.

File: steps/saucelabs-espresso-app-automate/0.1.4/step.yml [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+    title: SauceLabs access key
+  saucelabs_access_key: null
+- opts:
+    category: Automated Test Options
```
Comment (godrei) [inline steps/saucelabs-espresso-app-automate/0.1.4/step.yml]: Pls do not hide required inputs under a category

**Comment by godrei** [comment]
Hi @moatazeldebsy , thanks for the contribution, I found 2 issues with your step, pls fix them in a new StepLib PR.

**Pull request closed without merging**